scripts/deployment-test/netlify.mjs

```diff
@@ -27,6 +30,12 @@ async function createNewApp() {
     useTypeScript: true,
     projectDir: PROJECT_DIR,
   });
+
+  // In conjunction with NETLIFY_EDGE_CI we explicitly call remix init
+  // to create the edge version of the app
+  if (typeof NETLIFY_EDGE_CI !== "undefined") {
```

Comment:
I went with this because currently, none of the templates that deploy have a remix.init script.